drizzle/meta/0001_snapshot.json

```diff
@@ -1404,11 +1404,6 @@
       "foreignKeys": {},
       "compositePrimaryKeys": {},
       "uniqueConstraints": {
-        "VerificationToken_identifier_unique": {
```

Comment:
This is a weird phantom constraint that I cant see any other reference to. It might have been me doing some sketchy stuff earlier on in the migration but its unused and not needed so I have removed it from all the snapshots

The reason I removed it is because when I ask drizzle to generate a schema migration it looks at the snapshots and doesnt find this constraint in use anymore and trys to remove it. But it doesnt exist so this breaks the migration